erc-template.md

```diff
@@ -1,13 +1,13 @@
 ---
```

Comment:
```suggestion
---
eip: 7574
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please create a new file for the ERC rather than updating the template, i.e. erc-7574.md